docs/cloud/features/06_security/02_cloud-access-management/cloud-access-management.md

```diff
@@ -9,10 +9,15 @@ import Image from '@theme/IdealImage';
 import user_grant_permissions_options from '@site/static/images/cloud/security/cloud-access-management/user_grant_permissions_options.png';
 
 # Access control in ClickHouse Cloud {#access-control-in-clickhouse-cloud}
-ClickHouse controls user access in two places, via the console and via the database. Console access is managed via the clickhouse.cloud user interface. Database access is managed via database user accounts and roles. Additionally, console users can be granted roles within the database that enable the console user to interact with the database via our SQL console.
+
+ClickHouse controls user access in two places, via the console and via the database.
+Console access is managed via the clickhouse.cloud user interface. Database access is managed via database user accounts and roles.
```

Comment:
my initial reaction was clickhouse.cloud was a spellign mistake, then i realised you were refering to the domain. Normally we just use ClickHouse Cloud? any reason not to?